sys/linux/vnet.txt

```diff
@@ -836,6 +887,26 @@ tcp_header {
 	options		tcp_options
 } [packed]
 
+mptcp_header {
+    src_port    sock_port
+	dst_port	sock_port
+	seq_num		tcp_seq_num
+	ack_num		tcp_seq_num
+	ns		int8:1
+	reserved	const[0, int8:3]
+	data_off	bytesize4[parent, int8:4]
+	flags		flags[mptcp_flags, int8]
+	window_size	int16be
+	csum		csum[mptcp_packet, pseudo, IPPROTO_TCP, int16be]
+	urg_ptr		int16be
+	options		mptcp_options
+}
```

Comment:
This should be `[packed]`.

Reply:
Shouldn't this be deleted as it differs only in flags and options from tcp_header?

Reply:
See:
https://github.com/google/syzkaller/pull/848#issuecomment-443731070

Reply:
Yes. Right. So mptcp_header gets removed like mptcp_packet

Reply:
Right, missed that comment.